be/src/exec/pipeline/query_context.cpp

```diff
@@ -53,6 +53,13 @@ QueryContext::~QueryContext() noexcept {
     // remaining other RuntimeStates after the current RuntimeState is freed, MemChunkAllocator uses the MemTracker of the
     // current RuntimeState to release Operators, OperatorFactories in the remaining RuntimeStates will trigger
     // segmentation fault.
+    if (_mem_tracker != nullptr) {
```

Comment:
will this line bomb the be log file?

Reply:
Only one log will be printed per query (less than the audit log). I think it's acceptable.